way_to_home/utils/validators.py

```diff
@@ -246,3 +246,12 @@ def login_validator(data):
     if not required_keys_validator(data, required_keys):
         return False
     return True
+
+
+def profile_validator(data):
+    """Function that provides user_profile data validation"""
+    if not string_validator(data.get('first_name'), max_length=50):
+        return False
+    if not string_validator(data.get('last_name'), max_length=50):
+        return False
```

Comment:
I think you need to change validator to such as we did for place/route/etc with `validation_rules`, because your validator will return `False` if one of the fields is `None`. Also, note that the maximum length can be 64 for these fields.

Reply:
I agree, done.